teos/src/main.rs

```diff
@@ -64,11 +63,17 @@ fn create_new_tower_keypair(db: &DBM) -> (SecretKey, PublicKey) {
     db.store_tower_key(&sk).unwrap();
     (sk, pk)
 }
-
 #[tokio::main]
 async fn main() {
     let opt = Opt::from_args();
     let path = config::data_dir_absolute_path(opt.data_dir.clone());
+    let conf_file_path = path.join("teos.toml");
+
+    // Clone the conf_file_path variable before passing it to the from_file function
+    let cloned_conf_file_path = conf_file_path.clone();
+
+    // Log datadir path
+    log::info!("Using data directory: {:?}", &path);
```

Comment:
As mentioned by @sr-gi in the conversation below, the data directory information is stored in `path_network` variable instead of the path variable. Kindly look into it.

reference: https://github.com/talaia-labs/rust-teos/issues/202#issuecomment-1461553794

Reply:
Indeed. Given the tower can be run in multiple networks using the same root dir, the data dir needs to include the network path (check in the issue how `bitcoind` reports this, we're trying to follow the exact same pattern).

Reply:
> As mentioned by @sr-gi in the conversation below, the data directory information is stored in `path_network` variable instead of the path variable. Kindly look into it.
> 
> reference: [#202 (comment)](https://github.com/talaia-labs/rust-teos/issues/202#issuecomment-1461553794)

there is no path_network variable I have checked. It says to log the data drectory which is stored in path itself.

Reply:
> As mentioned by @sr-gi in the conversation below, the data directory information is stored in `path_network` variable instead of the path variable. Kindly look into it.
> 
> reference: [#202 (comment)](https://github.com/talaia-labs/rust-teos/issues/202#issuecomment-1461553794)

It's the default data directory